src/wallet/rpcwallet.cpp

```diff
@@ -4043,6 +4063,8 @@ static RPCHelpMan send()
                     {"include_unsafe", RPCArg::Type::BOOL, RPCArg::Default{false}, "Include inputs that are not safe to spend (unconfirmed transactions from outside keys and unconfirmed replacement transactions).\n"
                                                           "Warning: the resulting transaction may become invalid if one of the unsafe inputs disappears.\n"
                                                           "If that happens, you will need to fund the transaction with different inputs and republish it."},
+                    {"minconfs", RPCArg::Type::NUM, RPCArg::Default{0}, "If add_inputs is specified, require inputs with at least this confirmations."},
```

Comment:
It's singular:
https://github.com/bitcoin/bitcoin/blob/b295395664bd37e26d168c329f238237b34aef8c/src/wallet/rpcwallet.cpp#L2844-L2845